### DISCUSSION TITLE
FcRn binding competition

### DISCUSSION THREAD

**Discussion by RaphBil** [Open-Systems-Pharmacology/Forum#1551] [How-Tos] [answered]
Hello,

When modeling a concomitant administration of two large molecules with different affinity for FcRn receptor in endosomes, I was expecting to see an impact on the PK of the less potent one due to saturation or blocking of FcRn receptors by the potent compound administered at very high dose (to serve as a blocking agent).
However, simulations in PK-sim show no impact at all on the second compound whatever the simulated amounts and dosing intervals are.
Is there another way to model FcRn binding competition or saturation in PK-sim?

Thank you.
Best,
Raphael

**Comment by StephanSchaller** [comment, marked as answer]
Hi @RaphBil , we explored the use of high-dose Igg as a rescue treatment for high-affinity FcRn binders in the past, and this worked. Doses above 1 mg are needed, though, before FcRn is saturated sufficiently.
Happy to chat to discuss details.

Best,
Stephan

**Reply by wilbertdew** [reply]
hi @RaphBil, if there is high affinity in endosomes, you might also have some affinity in plasma. To incorporate the impact of those molecules at lower doses, you can modify the model a bit in MoBi as described here: https://pubmed.ncbi.nlm.nih.gov/36877385/
